### PR TITLE
ctrlr: init at 6.0.36

### PR DESCRIPTION
**Pull request by ghost**
###### Motivation for this change

Control your MIDI life (MIDI editor for all your hardware)

Instead of #81493 and #104396

cc @orivej @SuperSandro2000

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/ctrlr/default.nix [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{ stdenv, fetchFromGitHub, pkgconfig, alsaLib, curl, freetype, zenity
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/ctrlr/default.nix:1]:
```suggestion
{ stdenv, fetchFromGitHub, pkg-config, alsaLib, curl, freetype, zenity
```

File: pkgs/applications/audio/ctrlr/default.nix [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    rev = "2f9ce2ea74e49cb324442eae530b74aff59c35f2";
+    sha256 = "1hj15qms1dpi9v0qslsdqxq0kh659wh8ggzj9wym3mmw5nhxlwhq";
+    fetchSubmodules = true;
+  };
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [
+    alsaLib curl freetype libX11 libXext libXinerama gtk3 webkitgtk
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/ctrlr/default.nix:30]:
```suggestion
  nativeBuildInputs = [ pkg-config ];
  buildInputs = [
    alsaLib curl freetype libX11 libXext libXinerama gtk3 webkitgtk
```

Is curl required at runtime? if so it is at wrong place. A wrapper would probably be the right choice here.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 105047` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ctrlr</li>
  </ul>
</details>

File: pkgs/applications/audio/ctrlr/default.nix (hunk 2 above)
Comment (gnidorah) [reply pkgs/applications/audio/ctrlr/default.nix]: @SuperSandro2000 It links with libcurl

**Comment by SuperSandro2000** [comment]
Closing cause the author is a ghost user.

**Pull request closed without merging**